string grasp_name
---
StandardisedGrasp grasp_message
bool success
